hw/ip/i2c/data/i2c.hjson

```diff
@@ -493,5 +496,28 @@
         { bits: "7:0" }
       ]
     }
+    { name: "STRETCH_CTRL"
+      desc: "I2C target clock stretching control"
+      swaccess: "rw"
+      hwaccess: "hro"
+      fields: [
+        { bits: "0"
+          name: "EN"
+          desc: "Enable clock stretching after ongoing transaction completes"
+        }
+        { bits: "1"
+          name: "RESUME"
+          desc: "Stop clock stretching and resume normal operation"
+        }
+      ]
+    }
+    { name: "TIMEOUT_CTRL1"
```

Comment:
maybe we can just call this `host_timeout`?

Reply:
Renamed host_timeout_ctrl